app/build.gradle

```diff
@@ -32,11 +32,12 @@ android {
 }
 
 dependencies {
-
     implementation 'com.android.support:appcompat-v7:28.0.0'
```

Comment:
I think you should use the AndroidX libraries here. Android Studio also shows an error here for the legacy support library:
`
Version 28 (intended for Android Pie and below) is the last version of the legacy support library, so we recommend that you migrate to AndroidX libraries when using Android Q and moving forward. The IDE can help with this: Refactor > Migrate to AndroidX
`
if there's an issue with the automatic migration let me know, ideally just some dependencies and import statements need to be updated